rest/route/middleware.go

```diff
@@ -312,7 +313,22 @@ func RequiresProjectPermission(permission string, level evergreen.PermissionLeve
 		PermissionKey: permission,
 		ResourceType:  evergreen.ProjectResourceType,
 		RequiredLevel: level.Value,
-		ResourceFunc:  urlVarsToScopes,
+		ResourceFunc:  urlVarsToProjectScopes,
+	}
+	return gimlet.RequiresPermission(opts)
+}
+
+func RequiresDistroPermission(permission string, level evergreen.PermissionLevel) gimlet.Middleware {
```

Comment:
This could be combined with the function above, and have the resource type be passed in

Reply:
I think it is more readable this way (also the ResourceFuncs are different)